content/800-data-platform/200-projects/100-add-project/130-import-existing-project-new-db.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Import an existing project with a new database'
+metaTitle: 'Import an existing JavaScript/TypeScript project with a new database in the Prisma Data Platform'
+metaDescription: 'If you want to test how your JavaScript/TypeScript application is created and managed in the Prisma Data Platform, you can import your repository and connect the application to a new database. The Prisma Data Platform applies all schema migrations to the new database.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Import your application and use an empty database to explore and test project creation and the features in the Prisma Data Platform.
+
+</TopBlock>
+
+## Steps
+
+1. Use the steps provided in [Import an existing project with an existing database](/data-platform/projects/add-project/import-existing-project-existing-db) to import a project.
+2. At step 3.1, provide a connection string to an empty database.
+
+## Result
+
+The **Data Browser** opens with a message that the database schema contains no models.
+
+<img
+    src="https://res.cloudinary.com/prismaio/image/upload/v1666966856/docs/pdp/create-manage-projects/import-project-empty-database-result-database-schema-has-no-models_xzlubb.png"
+    alt="Prisma Data Platform - Import project with an empty database - Data Browser shows database schema contains no models"
+    width="750px"
+/>
+
+## What's next
+
+If you have existing migrations, you can use `migrate deploy` to [manually deploy your Prisma schema to the new database](/concepts/components/prisma-migrate/migrate-development-production#production-and-testing-environments).
+
+Alternatively, you can [create new database migrations with `migrate dev`](/concepts/components/prisma-migrate/get-started) and continue exploring the Prisma Data Platform.
```

Comment:
The `migrate` commands here omit the `prisma` prefix that is required to make them work. In this case, I would expect the `prisma` prefix to be included since no prior context about the command has been established (i.e. a reader landing on this page might not recognize that this command is supposed to be `prisma migrate` if they want to make it work) and it otherwise is incomplete.